app.py

```diff
@@ -35,6 +57,12 @@ def update_cfg_by_env(cfg):
     env_model_settings['base_path'] = os.getenv('S3_BASE_PATH')
     env_model_settings['access_key_id'] = os.getenv('S3_ACCESS_KEY')
     env_model_settings['secret_access_key'] = os.getenv('S3_SECRET_KEY')
+    env_model_settings['gpg_sign_key'] = os.getenv('GPG_SIGN_KEY')
+    # S3_GPG_SIGN_KEY stores GPG secret key for signing the repositories
+    # metadata. Our task is to add this key to the system and get its name.
+    if env_model_settings['gpg_sign_key']:
+        env_model_settings['gpg_sign_key'] = \
+            add_gpg_key(env_model_settings['gpg_sign_key'].encode('ascii'))
```

Comment:
My point as about the `GPG_SIGN_KEY` environemnt variable as well as about `gpg_sign_key` field in the `env_model_settings` dictionary. The meaning is either a key itself or its name depending of a context. We should not use the same term for different entities: it is confusing.

I would especiallly highlight it on the environment variables, which as part of a public contract in mkrepo and RWS:

* `GPG_SIGN_KEY` is a key itself for RWS.
* `GPG_SIGN_KEY` is a key name for mkrepo.

We should either use something like `GPG_SIGN_KEY_NAME` in mkrepo or use `GPG_SIGN_KEY_ARMORED` in RWS. Or other names, but different ones.

Reply:
`GPG_SIGN_KEY_ARMORED` - I like this one.

Reply:
`env_model_settings['gpg_sign_key']` still used for values with different meaning (key itself, key name).

Reply:
Ok. If I understand you correctly, I have updated this piece of code.

Reply:
You understood me correctly.